### DISCUSSION TITLE
Unable to install pip requirments

### DISCUSSION THREAD

**Discussion by iyengarm** [imartinez/privateGPT#320] [General]
<img width="1349" alt="image" src="https://github.com/imartinez/privateGPT/assets/3239288/0faf243c-1c22-478f-a262-d9cafb1cd311">

**Comment by imartinez** [comment]
Apparently there are some incompatibilities. I'd suggest you to create a python venv for the project, activate it and then install the dependencies there. That will ensure a clean installation and avoid incompatibilities. Remember to always activate the environment whenever you go back to the folder to execute privateGPT
